tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -258,6 +258,8 @@ impl EnvFilter {
             .collect();
 
         if !disabled.is_empty() {
+            // NOTE: this can't use `MakeWriter` because it's not yet available;
+            // there is not yet a subscriber.
```

Comment:
```suggestion
            // NOTE: We can't use a configured `MakeWriter` because the EnvFilter
            // has no knowledge of any underlying subscriber or collector, which 
            // may or may not use a `MakeWriter`.
```
(Terminology borrowed from #1015)